server/graphql/v2/object/MemberInvitation.ts

```diff
@@ -0,0 +1,64 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import { MemberRole } from '../enum/MemberRole';
+import { Account } from '../interface/Account';
+
+import { Tier } from './Tier';
+
+export const MemberInvitation = new GraphQLObjectType({
+  name: 'MemberInvitation',
+  description: 'An invitation to join the members of a collective',
+  fields: () => {
+    return {
+      id: {
+        type: GraphQLInt,
+        resolve(member) {
+          return member.id;
+        },
+      },
+      createdAt: {
+        type: GraphQLDateTime,
+        resolve(member) {
+          return member.createdAt;
+        },
+      },
+      collective: {
```

Comment:
We use the term `Account` on GQLV2

```suggestion
      account: {
```